.github/workflows/publish-typist-package-release.yml

```diff
@@ -82,38 +67,22 @@ jobs:
                   GIT_COMMITTER_NAME: Doist Bot
 
             - name: Remove Doist registry configuration from `.npmrc`
+              if: ${{ steps.semantic-release.outputs.package-published == 'true' }}
```

Comment:
This is either a valid question, or a learning opportunity for me, but where are these being set to true? I can't see anything setting them to be true, so will they always be false?

Reply:
`semantic-release` is governed by the `.releaserc` file, as the last step, when a release is successful, we change `package-published` to `true` ([ref](https://github.com/Doist/typist/pull/40/files#diff-644a8fae29c8e8b6ad5405a2e66c718a3b27f2e12581b7c5e00396d32b3e5cdeR32)).

Reply:
Ah nice, learning moment for me then :)